### ISSUE TITLE
Vscode prettier

### ISSUE THREAD

**Issue by massimoliani** [angel-dart/jael#44]
I'm building a server app with Jael rendering to protect the business logic as JavaScript framework can expose all the source code.
To simplify the development will be great to use the beautify as a t check the grammar too and allow to find quickly errors. I tried with html prettier but it change the {{- moving the minus near to the expression. If I use the comment  the parser probably skip a comment tag and give many errors on the html code following.

Can you help me?

Thank you.

PS. If you need the Jael file let me know.

**Comment by thosakwe** [comment]
As you can imagine, Jael is probably not compatible with the HTML prettier. There is work on a language server here: https://github.com/angel-dart/vscode/tree/master/jael_language_server

I haven't touched it in a while, but eventually I will roll the `JaelFormatter` class, and a `jaelfmt` tool, with the `jael` package itself.

**Comment by massimoliani** [comment]
Thanks.

I think the easiest way for me is to use your Jael function in angel_jael copying it and introducing just after file read a replacement of text. Like {{# to {{- for the escape and removal of comment to avoid Jael processor error. In this way I can force html beautify to see better the closing tags. 

In the future release can be interesting to introduce a parameter in the Jael function to process the file before the document parsing, as patch parameter apply after the document parsing.

**Comment by thosakwe** [comment]
Just released version `2.0.2`: https://pub.dev/packages/jael/versions/2.0.2
If there's additional functionality you would like to see, feel free to send in a pull request.

In the meantime, you can easily format a file:

```bash
pub global activate jael
jaelfmt -w . # Format all Jael files
```
